fetchai/agents-aea
# [AEA-529] Feature - Add p2p stub

### Changes

Implement a p2p stub connection based on the regular stub connection.
This will be useful for testing p2p protocols locally.

### How to test

Example https://github.com/fetchai/agents-p2p/tree/ec7d5c6c16b078410b22039060e3fa944ccaa863#p2p-stub-connection

Note: for pr history check https://github.com/fetchai/agents-aea/pull/1074
